scalableminds/webknossos
# fix dataset edit button shown to users who cant edit

### URL of deployed dev instance (used for testing):

### Steps to test:
1. open dataset and use the changed edit icon in the info tab, next to the dataset name
2. share the current dataset to a secondary account
3. verify that the icon does not appear in read-only mode

### Issues:
- fixes #5791 

------
- [✓] Ready for review
